docker/package/model.py

```diff
@@ -300,6 +312,37 @@ def gen_spec_file(self, build_deps, run_deps, out):
         ) = gen_spec_systemd_part(self)
         version = self.meta.version.replace("-", "")
 
+        additional_files_install = (
+            "\n".join(
+                map(
+                    lambda x: f"install -m 0755 %{{_sourcedir}}/{x} %{{buildroot}}/%{{_bindir}}\n"
+                    + f'ln -s %{{_bindir}}/{x} %{{buildroot}}/%{{_bindir}}/{x.replace("octez", "tezos")}'
+                    if "octez" in x
+                    else "",
+                    self.additional_files,
+                )
+            )
+            if self.additional_files
+            else ""
+        )
+
+        additional_files = (
+            "\n".join(
+                map(
```

Comment:
`map(lambda ..., <list>` can be replaced with list comprehension (`["%{bindir}/" + x for x in self.additional_file + ...]`}. Thus it'll be a bit more readable, I believe